README.md

```diff
@@ -79,6 +79,19 @@ config :mix_test_watch,
   clear: true
 ```
 
+## Excluding files or directories
+
+To ignore changes from specific files or directories just add `exclude:` patterns
+to your config in `mix.exs`:
+
+```elixir
+config :mix_test_watch,
+  exclude: ["db_migration/.*", "useless_.*\.exs"]
+```
+
+It accepts either regexp struct `~r/\w+/` or pattern as string `"\w+"` which
```

Comment:
My intention was to make the configuration flexible, simple and intuitive.  From my standpoint there is no big difference in semantic `"*pattern*"` vs `~r/*pattern*/` for cinfiguration, but string pattern is more convenient  for human beings :) I understand that there is a big difference between those in runtime.